lambda/textractor/python/og.py

```diff
@@ -29,6 +30,52 @@ def prune_blocks(o):
 
     return o
 
+def format_date(date):
+    try:
```

Comment:
Instead of having a long list of nested except clauses, there are 2 alternate approaches that look cleaner.
You can iterate over the list of acceptable date formats and return the date_object whenever no error is raised from the code.
You can also explore the datefinder module and see if its feasible to use that.
You can find both those approaches here : https://stackoverflow.com/questions/9507648/

Reply:
That is a good idea. I found the iterative method more suitable and will make changes accordingly.